src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -157,6 +157,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('values', help="The task values file path relative to the source context.")
         c.argument('set_value', options_list=['--set'], help="Value in 'name[=value]' format. Multiples supported by passing --set multiple times.", action='append', validator=validate_set)
         c.argument('set_secret', help="Secret value in '--set name[=value]' format. Multiples supported by passing --set multiple times.", action='append', validator=validate_set_secret)
+        c.argument('agent_pool_name', options_list=['--agent-pool'], help='The name of the agentpool.')
```

Comment:
later, can you add a preview flag for the agent_pool_name? Also mark the agent_pool command group as preview.

Reply:
done